server.js

```diff
@@ -42,15 +42,6 @@ if (fs.existsSync('.env')) {
   process.exit(1)
 }
 
-if (config.private.redis_url != null) {
```

Comment:
The decision I have taken here is just to remove `redis_url` from the config schema.
The other thing we could do would be leave it in the schema, keep this line here and throw an exception if `redis_url` is set.

Reply:
Long term I don't think it's necessary to have that, though I find myself somewhat ambivalent about what to do in the short term. I could see value in having the exception explicitly thrown for another monthly release or two